pom.xml

```diff
@@ -352,7 +348,14 @@
             </build>
         </profile>
     </profiles>
-    <!-- For Spring Bundlor plugin -->
+
+    <repositories>
+        <repository>
+            <id>spring-libs-release</id>
+            <url>https://repo.spring.io/libs-release</url>
```

Comment:
Why do you need this one?

Reply:
We don't really _need_ it. Just aligned it with other _spring-data-*_ projects to make it easier to take over had Spring decided to do so

Reply:
I'd get rid of that. It's unlikely Spring will take over this project any time soon.